control/src/pages/contact-us/components/contactFormContainer/contactFormContainer.js

```diff
@@ -4,6 +4,7 @@ import detailsData from './detailsArray'
 import axios from 'axios'
 import { useDropzone } from 'react-dropzone'
 import Alert from '../../assets/alert-circle.svg'
+import { GetUserInfo } from '../../../../zuri-control'
```

Comment:
@pidoxy is this the right import or "@zuri/control"?

Reply:
This is correct as well since it's in the same repo

Reply:
alright